mcm/deploy/action.sh

```diff
@@ -7,10 +7,22 @@
 echo "Docker version $(docker version)"
 echo "Docker Compose version $(docker compose version)"
 
+# Download and decompress the required data
+if [ -z "$MCM_EXAMPLE_DATA_URL" ]; then
+    echo 'Set $MCM_EXAMPLE_DATA_URL with the URL for downloading the McM data'
+    exit 1
+fi
+
+echo 'Downloading McM data....'
+MCM_DATA_FILE="$HOME/data.tar.gz"
+curl -s -o $MCM_DATA_FILE $MCM_EXAMPLE_DATA_URL
+echo 'Decompressing ....'
+tar -xf $MCM_DATA_FILE -C $HOME/
```

Comment:
I think this is more idiomatic
```suggestion
curl -s $MCM_EXAMPLE_DATA_URL | tar -xC $HOME
```

Reply:
Doesn’t this raise issues in case one chunk of the file is downloaded improperly? Or does this download the complete file first, store it in memory, and then decompress it?

Reply:
@lmoureaux...

Reply:
It's no different than storing the archive on disk and decompressing it later

Reply:
Okay, let's include it